src/components/pages/Deals.js

```diff
@@ -1,11 +1,49 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import Item from '../items/Item';
+
+const Deals = (props) => {
+  const { items } = props;
+  // filtering for items on sale
+  const deals = items.filter((item) => item.isOnSale);
```

Comment:
The deals should be filtered from the API

Reply:
fixed in a9af71b2e0c082692b9402817ed3a70cd11571db